src/services/rpcTransactionService.ts

```diff
@@ -29,6 +30,12 @@ export class RpcTransactionService {
     }
 
     public addTransaction(transaction: Transaction) {
-        this.client.addTransaction(transaction);
+        this.client.addTransaction(transaction, (err: Error) => {
+            if (err) {
```

Comment:
Please change this if..else statement to the following:
`if (err) {
    logger.error(err);
    return;
}
logger.info("Transaction sending succeeded");`